bootstrap3-typeahead.js

```diff
@@ -61,7 +61,8 @@
     this.followLinkOnSelect = this.options.followLinkOnSelect || this.followLinkOnSelect;
     this.source = this.options.source;
     this.delay = this.options.delay;
-    this.$menu = $(this.options.menu);
+    this.theme = this.option.theme && this.options.themes && this.options.themes[this.options.theme] || Typeahead.defaults.themes[Typeahead.defaults.theme];
```

Comment:
@bassjobsen @kristofdegrave this seems to be a breaking change: `this.option` is undefined (missing letter `s`)

Reply:
It should be `this.options` type (sorry :) )

Reply:
Added PR #336